src/main/public/js/append-row.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(function () {
+  $('input[name="action[addRow]"]').click(function () {
+    var lastRow = $('.claim-amount-row, .multiline-row').last()
+    var parent = lastRow.parent()
+    var newRow = lastRow.clone()
```

Comment:
Will that clone errors as well? For example when amount was invalid on last row prior adding new row?

Reply:
Yes, it will 😞 Good catch.

Reply:
That makes the matter a bit more problematic.